src/template/ProjectArticle.tsx

```diff
@@ -85,6 +96,25 @@ export const query = graphql`
           slug
         }
         linkGithub
+        corsi {
+          titolo
+        }
+      }
+    }
+    nextProject: allContentfulProgetti(
```

Comment:
Userei la query che prende solo un progetto tipo per il prossimo progetto, tipo cosi. 
`
{
  contentfulProgetti(
    corsi: {elemMatch: {idCorso: {eq: "3192370"}}}
    ordine: {eq: 2}
  ) {
    titolo
  }
}
`

In realtà anche per prendere il progetto dell'articolo sarebbe meglio utilizzare la query che prende solo 1 progetto  così ti levi dai coglioni anche robe di nodes e array dispettosi